components/translation/usage.rst

```diff
@@ -166,29 +166,23 @@ use for translation::
         'fr_FR'
     );
 
-    $translator->transChoice(
+.. _retrieving-the-message-catalogue:
+
+Pluralized Translations
+-----------------------
+
+Translation messages can be pluralized using `interval notations`_, the count
```

Comment:
where the counter value to choose the right translation can be provided using

Wdyt?

Reply:
I've removed this section based on this comment: https://github.com/symfony/symfony-docs/pull/12957#pullrequestreview-344998844

Reply:
I've removed this section based on this comment: https://github.com/symfony/symfony-docs/pull/12957#pullrequestreview-344998844

Reply:
I've removed this section based on this comment: https://github.com/symfony/symfony-docs/pull/12957#pullrequestreview-344998844